R/copy_no_clobber.R

```diff
@@ -1,3 +1,13 @@
+#' @title Copy not clobber.
```

Comment:
Is this used anywhere that we know of?

Reply:
This is the only place I found it being used: https://github.com/morinlab/GAMBLR.results/blob/main/R/check_times.R